sdk/storage/azure-storage-file-share/azure/storage/fileshare/_directory_client.py

```diff
@@ -10,7 +10,7 @@
 import time
```

Comment:
**Note:** The imports has not been fully reordered since there's still a small `mypy` error, will reorder them when I check in the next set of changes :)